controller/destination/listener.go

```diff
@@ -26,6 +28,10 @@ type updateAddress struct {
 	pod     *coreV1.Pod
 }
 
+func (ua updateAddress) String() string {
+	return fmt.Sprintf("{address:%v, pod:%s.%s}", ua.address, ua.pod.Namespace, ua.pod.Name)
```

Comment:
Since `ua.address` is the integer representation of an IP address, we probably could convert it to a more readable version using `addr.ProxyAddressToString(...)` The "toString" function would look like this:

```go
func (ua updateAddress) String() string {
	return fmt.Sprintf("{address:%v, pod:%s.%s}", addr.ProxyAddressToString(ua.address), ua.pod.Namespace, ua.pod.Name)
}
```

With this version, the log lines will end up looking like this:

```bash
Updating emoji-svc.emojivoto:8080 to [{address:10.1.0.178:8080, pod:emojivoto.emoji-6b9cfb4f46-pnl92}]
```